src/libraries/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -48,48 +49,29 @@
     <Compile Include="System\Net\Sockets\OverlappedAsyncResult.cs" />
     <Compile Include="System\Net\Sockets\ReceiveMessageOverlappedAsyncResult.cs" />
     <Compile Include="System\Net\Sockets\UnixDomainSocketEndPoint.cs" />
-    <Compile Include="$(CommonPath)System\IO\StreamHelpers.CopyValidation.cs"
-             Link="Common\System\IO\StreamHelpers.CopyValidation.cs" />
+    <Compile Include="$(CommonPath)System\IO\StreamHelpers.CopyValidation.cs" Link="Common\System\IO\StreamHelpers.CopyValidation.cs" />
```

Comment:
Could you revert the unnecessary csproj changes? I.e. keep the element split over 2 lines.